lm75bd/lm75bd.c

```diff
@@ -27,7 +27,23 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
+  uint8_t buf[2];
+
+  uint8_t addr = 0x0U;
+  i2cSendTo(devAddr, &addr, 1);
+
+  i2cReceiveFrom(devAddr, buf, 2);
```

Comment:
I would use sizeof(buf) instead of hardcoding 2 so that if buf size ever changes this doesn't need to be changed as well